packages/ckeditor5-package-generator/bin/index.js

```diff
@@ -24,7 +24,8 @@ new Command( packageJson.name )
 		// Otherwise, the executable was downloaded from npm, and it can't be executed in dev-mode.
 		return fs.existsSync( path.join( rootRepositoryPath, '.git' ) );
 	}, false )
-	.option( '--use-npm', 'whether use npm to install packages', false )
+	.option( '--use-npm', 'whether use npm to install packages' )
+	.option( '--use-yarn', 'whether use yarn to install packages' )
```

Comment:
Let's set default values for both `--use-npm` and `--use-yarn` flags to `false`, so the `choosePackageManager()` does not have to set these default values manually.